src/main/java/core/basesyntax/service/WritingToNewFile.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WritingToNewFile {
+    void writing(String fileName);
```

Comment:
The method name 'writing' could be more descriptive. Consider renaming it to something like 'writeToFile' to clearly indicate its purpose.